gcp_variant_transforms/testing/integration/run_tests_common.py

```diff
@@ -40,43 +54,71 @@ class TestCaseFailure(Exception):
 class TestRunner(object):
   """Runs the tests using pipelines API."""
 
-  def __init__(self, tests):
-    # type: (List[Union[PreprocessorTestCase, VcfToBQTestCase]]) -> None
+  def __init__(self, tests, revalidation=None):
+    # type: (List[List[TestCaseInterface]], str) -> None
+    """Initializes the TestRunner.
+
+    Args:
+      tests: All test cases.
+      revalidation: If not none, skip most of the test (including table creation
```

Comment:
Consider rephrasing as "skip most of the test" implies skipping actual test cases. Something like:
`revalidate: If True, only run the result validation part of the tests.`